lib/rubocop/options.rb

```diff
@@ -34,7 +34,7 @@ def parse(command_line_args)
     private
 
     def args_from_file
-      if File.exist?('.rubocop')
+      if File.file?('.rubocop')
```

Comment:
It does actually work for symbolic links. 
I find the `rescue` solution a bit less clear... plus it hides potential error messages!
Is it worth it for named pipes?
Maybe `File.exists?('.rubocop') && !File.directory?('.rubocop')`? or is that overkill?